src/ui/Analytics/Analytics.ts

```diff
@@ -291,8 +291,8 @@ export class Analytics extends Component {
    * @param element The HTMLElement that the user has interacted with for this custom event. Default value is the
```

Comment:
Should the documentation be updated to explain that the result parameter exists but is optional ?